inspire/modules/deposit/forms.py

```diff
@@ -463,24 +463,28 @@ class LiteratureForm(WebDepositForm):
                                              mapper=journal_title_kb_mapper)
     )
 
-    page_range_article_id = fields.TextField(
+    journal_page = fields.TextField(
         label=_('Page Range/Article ID'),
         description=_('e.g. 1-100'),
+        export_key='page_range_article_id',
         widget_classes="form-control" + ARTICLE_CLASS
```

Comment:
OK, great. Please, review the rest of the ``export_key`` so that we can directly map with the names in ``inspire.cfg`` and get rid of some code in ``process_sip_metadata``.

For the mapping [there](https://github.com/pedrogaudencio/inspire-next/blob/arxiv-split-pub-note/inspire/modules/deposit/views.py#L47) hopefully the API will be stable and we will receive always the same names, so should be ok.